NameGame/NameGame/Model/Headshot.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Headshot.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/13/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+struct Headshot {
+    
+    let type: String
+    let mimeType: String
+    let id: String
+    let url: String
+    let alt: String
+    let height: Int
+    let width: Int
+    
+    private enum JSONKey: String {
```

Comment:
Style change; move the enum definition above the properties. Also for future compatibility with Swift 4 lets change the name from `JSONKey` to `CodingKeys`.